core/src/mindustry/entities/bullet/BulletType.java

```diff
@@ -144,6 +144,12 @@ public boolean collides(Bullet bullet, Building tile){
     }
 
     public void hitTile(Bullet b, Building tile, float initialHealth){
+        if(status == StatusEffects.burning) {
+            Tile buildTile = world.tileWorld(tile.x, tile.y);
```

Comment:
...same here, tile can't be null, so use `tile.tile`

Reply:
Changed to `Fires.create(tile.tile);`